demos/crossroad_camera_demo/cpp/main.cpp

```diff
@@ -351,46 +383,83 @@ struct PersonAttribsDetection : BaseDetection {
         return hasTopBottomColor;
     }
 
-    InferenceEngine::CNNNetwork read(const InferenceEngine::Core& ie) override {
+    std::shared_ptr<ov::Function> read(const ov::runtime::Core& core) override {
         /** Read network model **/
-        auto network = ie.ReadNetwork(FLAGS_m_pa);
+        auto network = core.read_model(FLAGS_m_pa);
         /** Extract model name and load it's weights **/
-        network.setBatchSize(1);
+        const ov::Layout layout_nhwc{ "NHWC" };
+        ov::Shape input_shape = network->input().get_shape();
+        input_shape[ov::layout::batch_idx(layout_nhwc)] = 1;
+        network->reshape({ {network->input().get_any_name(), input_shape} });
+
         // -----------------------------------------------------------------------------------------------------
 
-        /** Person Attribs network should have one input two outputs **/
+        /** Person Attribs network should have one input and one or three outputs **/
         // ---------------------------Check inputs ------------------------------------------------------
-        InferenceEngine::InputsDataMap inputInfo(network.getInputsInfo());
-        if (inputInfo.size() != 1) {
+        ov::OutputVector inputs = network->inputs();
+        if (inputs.size() != 1) {
             throw std::logic_error("Person Attribs topology should have only one input");
         }
-        InferenceEngine::InputInfo::Ptr& inputInfoFirst = inputInfo.begin()->second;
-        inputInfoFirst->setPrecision(InferenceEngine::Precision::U8);
+
         if (FLAGS_auto_resize) {
-            inputInfoFirst->getPreProcess().setResizeAlgorithm(InferenceEngine::ResizeAlgorithm::RESIZE_BILINEAR);
-            inputInfoFirst->getInputData()->setLayout(InferenceEngine::Layout::NHWC);
+            network = PrePostProcessor(network).
+                input(InputInfo().
+                    tensor(InputTensorInfo().
+                        set_element_type(ov::element::u8).
+                        set_spatial_dynamic_shape().
+                        set_layout({ "NHWC "})).
+                preprocess(PreProcessSteps().
+                    convert_element_type(ov::element::f32). // WA for CPU plugin
+                    convert_layout("NCHW"). // WA for CPU plugin
```

Comment:
I think we need to remove these WAs for CPU and keep only resize

Reply:
removed